modules/core/src/test/java/org/apache/ignite/platform/PlatformCacheAffinityVersionTask.java

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
I do not understand changes in .net part.
They should be undone or clearly explained.

Reply:
I called Pavel, for checking .NET